platform/ios/docs/guides/Runtime Styling.md

```diff
@@ -0,0 +1,57 @@
+# Runtime Styling
+
+Mapbox's runtime styling features allow you direct control over every layer in your maps with code. It's now possible create dynamic maps and visualizations that aren't possible with other mobile maps SDKs.
```

Comment:
Typo: now possible **to** create.

Also, “mobile map SDKs” sounds slightly better.